.github/pull_request_template.md

```diff
@@ -35,6 +35,22 @@ _Please choose one of the keywords above to refer to the issue this PR solves fo
 
 _Optional. Remove if not applicable._
 
+## Documentation
+
+#### Requires documentation change?
+
+- [ ] Yes
+- [ ] No
+
+_Link to documentation or issue about change which needs to be added with "docs" label (example: https://github.com/NethermindEth/nethermind/issues/5856)_
```

Comment:
This message should be more generic as if there's an issue for the PR, it's already mentioned at the very top and not necessarily should have the `docs` label. I'll commit the suggested change shortly.